app_users/models.py

```diff
@@ -90,23 +94,10 @@ class AppUser(models.Model):
     display_name = models.TextField("name", blank=True)
     email = models.EmailField(null=True, blank=True)
     phone_number = PhoneNumberField(null=True, blank=True)
-    balance = models.IntegerField("bal")
     is_anonymous = models.BooleanField()
     is_disabled = models.BooleanField(default=False)
     photo_url = CustomURLField(default="", blank=True)
 
-    stripe_customer_id = models.CharField(max_length=255, default="", blank=True)
-    is_paying = models.BooleanField("paid", default=False)
-
-    low_balance_email_sent_at = models.DateTimeField(null=True, blank=True)
-    subscription = models.OneToOneField(
-        "payments.Subscription",
-        on_delete=models.SET_NULL,
-        related_name="user",
-        null=True,
-        blank=True,
-    )
-
```

Comment:
just grouped all the billing fields together for niceness below. these will be migrated to org